benchmark/ParseBenchmark.cs

```diff
@@ -0,0 +1,53 @@
+using BenchmarkDotNet.Attributes;
+using GrokNet;
+
+namespace Benchmark
+{
+    public class ParseBenchmark

```

Comment:
Could you please add MemoryDiagnoser? https://benchmarkdotnet.org/articles/configs/diagnosers.html